users/vieting/experiments/switchboard/ctc/feat/network_helpers/specaug_configurable.py

```diff
@@ -106,13 +119,66 @@ def add_specaug_layer(
     # Update config with pre-generated parameters
     full_config["specaug_params"] = specaug_params
 
+
     network[name] = {
         "class": "eval",
         "from": from_list,
         "eval": f"self.network.get_config().typed_value('transform')(source(0, as_data=True), network=self.network, **{full_config})",
     }
+    return [name], [
+        sort_filters_by_center_freq,
+        get_frequency_response,
+        filter_based_masking,
```

Comment:
Don't you already have experiments running with `specaug_configurable.py`? If you now always add this function, their hashes will break.

Reply:
This is correct, but not a big Problem, since the entire file is quite new and there are not a lot of experiments that use the new SpecAugment yet.

Reply:
Changed it